app/build.gradle

```diff
@@ -65,6 +65,7 @@ dependencies {
 
     implementation libraries.dagger
     implementation libraries.daggerSupport
+    implementation 'com.android.support.constraint:constraint-layout:1.0.2'
```

Comment:
It is already added as dependency at https://github.com/divyanshub024/Rocket.Chat.Android/blob/530ff7c7439822851dcdeaaa5df14bc7e475bb08/app/build.gradle#L60

Reply:
Everything else looks good to me :+1:

Reply:
Thanks, @Shailesh351